src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -16,6 +18,7 @@
  */
 public class AddressBook implements ReadOnlyAddressBook {
 
+    private final UniqueHealthWorkerList healthWorkers;
```

Comment:
Would it be a better idea to create a seperate class for storage of `UniqueHealthWorkerBook`? Since both objects serialize to JSON slightly differently

Reply:
Hmm that could be possible! Due to the focus on making  sure the command works first, I made minimal changes to addressbook, so there wasn't alot of focus on the structure of storage.